public/locales/en-US/send.ftl

```diff
@@ -19,66 +19,70 @@ notifyUploadDone = Your upload has finished.
 
 uploadingPageMessage = Once your file uploads you will be able to set expiry options.
 uploadingPageCancel = Cancel upload
-  .title = {uploadingPageCancel}
+  .title = Cancel upload
 uploadCancelNotification = Your upload was cancelled.
 
 uploadingPageLargeFileMessage = This file is large and may take awhile to upload. Sit tight!
 uploadingFileNotification = Notify me when the upload is complete.
 uploadSuccessConfirmHeader = Ready to Send
 
-uploadSvgAlt.alt = Upload
-
-
-//Note the spec suggests that this string is editable. That feature will not appear at Launch
+uploadSvgAlt
+    .alt = Upload
 uploadSuccessTimingHeader = The link to your file will expire after 1 download or in 24 hours.
 copyUrlFormLabel = Copy and share the link to send your file:
 copyUrlFormLabelWithName = Copy and share the link to send your file: { $filename }
+
 // Note: Title text for button should be the same
 copyUrlFormButton = Copy to clipboard
-  .title = {copyUrlFormButton}
+    .title = Copy to clipboard
 
 copiedUrl = Copied!
 
 deleteFileButton = Delete file
-  .title = {deleteFileButton}
+    .title = Delete file
 sendAnotherFileLink = Send another file
-  .title = {sendAnotherFileLink}
+    .title = Send another file
 
-downloadAltText.alt = Download
+downloadAltText
+    .alt = Download
 downloadFileName  = Download { $filename }
 downloadFileSize = ({ $size })
```

Comment:
The file size comes in from the server and is rendered through handlebars to the ftl file. It's a string in a human readable format, i.e. 12 Mb.